### PR TITLE
cmake-format

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

I've manually tried to make one too many cmake files look consistent.  I'm happy to make PRs to apply this in all our repos.

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)

### CODE REVIEW

File: moveit_core/CMakeLists.txt [hunk 1]
```diff
@@ -48,8 +48,7 @@ find_package(trajectory_msgs REQUIRED)
 find_package(visualization_msgs REQUIRED)
 find_package(common_interfaces REQUIRED)
 find_package(pluginlib REQUIRED)
-# TODO: Port python bindings
-# find_package(pybind11 REQUIRED)
+# TODO: Port python bindings find_package(pybind11 REQUIRED)
```
Comment (v4hn) [inline moveit_core/CMakeLists.txt:51]: not a nice cleanup

File: moveit_core/CMakeLists.txt [hunk 2]
```diff
@@ -78,12 +77,10 @@ set(THIS_PACKAGE_INCLUDE_DIRS
     planning_request_adapter/include
     planning_scene/include
     profiler/include
-    # TODO: Port python bindings
-    # python/tools/include
+    # TODO: Port python bindings python/tools/include
```
Comment (v4hn) [inline moveit_core/CMakeLists.txt:80]: same here and possibly in other places.

File: moveit_core/CMakeLists.txt [hunk 3]
```diff
@@ -102,59 +99,61 @@ set(THIS_PACKAGE_LIBRARIES
...
+    srdfdom
+    std_msgs
+    tf2_eigen
+    tf2_geometry_msgs
+    trajectory_msgs
+    visualization_msgs
+    Eigen3
+    eigen3_cmake_module
+    OCTOMAP
+    Bullet)
```
Comment (v4hn) [inline moveit_core/CMakeLists.txt:131]: Personally I prefer the `)` in a separate line, but I see how that is bike-shedding

File: moveit_core/CMakeLists.txt [hunk 4]
```diff
@@ -184,44 +183,34 @@ add_subdirectory(kinematics_metrics)
...
 install(
   TARGETS ${THIS_PACKAGE_LIBRARIES}
   EXPORT export_${PROJECT_NAME}
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
-)
+  INCLUDES
+  DESTINATION include)
```
Comment (v4hn) [inline moveit_core/CMakeLists.txt:213]: Is that a bug in the formatter or do the few characters make the line too long? :confused:

File: moveit_core/ConfigExtras.cmake [hunk 5]
```diff
@@ -1,6 +1,8 @@
 # Extras module needed for dependencies to find boost components
 
-find_package(Boost REQUIRED
+find_package(
+  Boost
+  REQUIRED
```
Comment (v4hn) [inline moveit_core/ConfigExtras.cmake:5]: Having the `REQUIRED` *anywhere* but right behind the package name seems off to me.

**Review by v4hn — COMMENTED** [review]
just some feedback, browsing through the changes.

Are you sure all these change requests on formatting you get on every single one of your recent PRs to MoveIt(2) is worth spending so much time on agreeing to automated formatting? :shark: 
It's not like we don't have substantial problems to tackle... Sorry for the remark :sunglasses:

**Comment by tylerjw** [comment]
@v4hn I see this one didn't do a nice job... that being said... I really want much more automated formatting so we can stop bikeshedding over formatting or manually doing it.  I caught myself trying to clean up and standardize style in cmake files today and that made me sad.

I remember working on a project that used Bazel and it had a formatted for the build files and it was really nice that they were all standardized.

**Comment by tylerjw** [comment]
> is worth spending so much time on agreeing to automated formatting? shark

I don't know if you've noticed but my approach to these formatting changes has been fairly low effort.

**Comment by vatanaksoytezer** [comment]
@tylerjw if we are happy with `ament_lint_cmake`s formatting I can quickly write a pre-commit hook to enforce that.

**Comment by v4hn** [comment]
> I don't know if you've noticed but my approach to these formatting changes has been fairly low effort.

I noticed, but they ask for quite a bit of review time. ;)

**Comment by tylerjw** [comment]
That is fair, I'll back off these for now.  Idk if you saw I made a PR with an api change to robot state and robot model.  I have a second version of that I'm hoping to open tonight when I finish it and present them at the meeting.  Basically I don't think returning nullptr is a nice interface for users and is easy to create code that segfaults.

**Pull request closed without merging**